core/src/main/java/org/arquillian/smart/testing/configuration/Configuration.java

```diff
@@ -155,9 +155,13 @@ public static Configuration load(File projectDir) {
             logger.info("Config file `" + SMART_TESTING_YAML + "` OR `" + SMART_TESTING_YML + "` is not found. "
                 + "Using system properties to load configuration for smart testing.");
         } else {
-            try (InputStream io = Files.newInputStream(getConfigurationFilePath(files))) {
+            final Path filePath = getConfigurationFilePath(files);
+            try (InputStream io = Files.newInputStream(filePath)) {
                 final Yaml yaml = new Yaml();
                 yamlConfiguration = yaml.load(io);
+                if (yamlConfiguration == null) {
```

Comment:
What's happening if file is empty but there are system properties set? Should we fail? Maybe the file is empty because someone else is the responsible of configuring.

Reply:
I am just thinking loudly

Reply:
I would expect that if the file is empty then we should not fail, just load the default values or those taken from system properties.
The only thing what we can do in this case is just logging the information/warn that the file is empty.

Reply:
I would log warning too to be honest. I guess that's what we do with empty `arquillian.xml` for example? Only fail when no sysprops are defined as well.

Reply:
Not sure about `arquillian.xml` but I guess that just warning is ok in this case. I agree with @MatousJobanek approach.